content/events/community_autumn_24/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "[Upcoming Event] Meeting ESP komunity - podzimní edice 2024"
+date: 2024-08-28
+draft: false
+description: "Espressif Community Meeting - Autumn 24"
+tags: ["event", "announcement", "community meeting", "czech transcript only"]
+showDate : false
+---
+
+{{< badge >}}
+New event!
+{{< /badge >}}
+
+## Nový Event se blíží
+
+Máme skvělou zprávu! Po úspěšných dvou ročnících tě zveme na naše třetí setkání československého bastlířského eventu, který se letos koná na novém místě – v areálu Vlněna, jen kousek od naší české kanceláře. Jestli tě baví bastlení, programování, zkrátka cokoli kolem našich ESPéček, nebo si prostě zvědavý na naše novinky, tohle si nesmíš nechat ujít!
+
+Těšit se můžeš na zajímavé přednášky, praktické dema, workshop, kde si můžeš vyzkoušet programování v ESP-IDF, a hlavně na skvělou atmosféru mezi lidmi, kteří stejně jako ty milují technologie a objevování nových možností. Přijď si užít večer plný inspirace a setkat se s dalšími nadšenci!
+
+
+### Na co se tedy můžeš těšit:
+
+<article class="gallery">
+  <img src="gallery/RAD_0032.jpg" />
+  <img src="gallery/RAD_0051.jpg" />
+  <img src="gallery/RAD_0060.jpg" />
+  <img src="gallery/RAD_0073.jpg" />
+  <img src="gallery/RAD_0078.jpg" />
+  <img src="gallery/RAD_0503.jpg" />
+  <img src="gallery/RAD_0526.jpg" />
+  <img src="gallery/RAD_0542.jpg" />
+</article>
+
+
+- **Přednášky** Čekají tě inspirativní přednášky od vývojářů z Espressif, lokálních firem a zapálených nadšenců do ESP! Přijď si poslechnout nejnovější trendy, tipy a triky přímo od lidí, kteří tvoří budoucnost těchto technologií.
+
+- **Workshop** Připravili jsme pro tebe dvouhodinový praktický workshop, kde si budeš moci osahat vývoj na našem novém čipu **ESP32C6**. Budeš mít jedinečnou příležitost ponořit se do vývoje s podporou odborníků! Nezapomeň si vzít **vlastní notebook** a pokud máš, přibal si s sebou kabel USB-A - USB-C.

```

Comment:
... na našem novém čipu **ESP32-C6** v prostředí ESP-IDF. Důraz je kladen na snadný přechod od jednodušších frameworků, jako Arduino nebo MicroPython. ... ... kabel s USB-C koncovkou.

Reply:
+ **ESP32C6** (Tento čip (devkit?) si po workshopu budeš moci nechat.)

Reply:
Opravdu si ho budou moct nechat? Toto bych bych zkonzultoval s Honzou. Mam dojem ze je rozdavat nechtel s tim ze by se mohli pak pouzit na workshopy jinde

Reply:
Ano, rozdávali jsme je i v Brazílii. Je to dobrý tahák pro účast workshopu.